packages/notifi-dapp-example/src/app/page.tsx

```diff
@@ -52,6 +70,67 @@ export default function Home() {
           <PoweredByNotifi />
         </div>
       </div>
+
+      {isOpenMobileReminderModal ? (
```

Comment:
This section looks kind of big to be here. Do you think it will be a good idea to move it to a new component?